A3A/addons/core/functions/CREATE/fn_createAttackForceMixed.sqf

```diff
@@ -112,12 +113,28 @@ if (_airBase != "") then            // uh, is that a thing
     };
     private _troops = ["Normal", "SpecOps"] select ("specops" in _modifiers);
     ServerDebug_3("Attempting to spawn %1 air vehicles including %2 attack from %3", _airCount, _attackCount, _airbase);
+    if (typeOf _vehicleType == "vehiclesDropPod") then {
```

Comment:
Does _vehicleType exist in this context?

Also are you sure you should be comparing to a literal string? Shouldn't you be comparing to `FACTION get "vehiclesDropPod"`